cmd/bootstrapper/initialize_kubelet.go

```diff
@@ -41,6 +41,8 @@ var (
 		clusterDNS string
 		// platformType contains type of the platform where the cluster is deployed
 		platformType string
+		// dockerRuntime indicates if the container runtime to be used is docker
```

Comment:
Can you pls add an extra line saying:
If true docker will be the runtime, false means containers will be the runtime.